public/pages/cadastro/cadastro_pais.html

```diff
@@ -35,36 +35,36 @@
           <b>
           <ul class="nav">
             <li class="nav-item">
-              <a class="nav-link navbar-link" href="/public/pages/about.html"> QUEM SOMOS </a>
+              <a class="nav-link navbar-link" href="./public/pages/about.html"> QUEM SOMOS </a>
             </li>
             <li class="nav-item">
-              <a class="nav-link navbar-link" href="/public/pages/feed.html"> FEED </a>
+              <a class="nav-link navbar-link" href="./public/pages/feed.html"> FEED </a>
             </li>
             <li class="nav-item dropdown">
               <a class="nav-link navbar-link dropdown-toggle" data-bs-toggle="dropdown" href="#" role="button"
                 aria-expanded="false">ATIVIDADES</a>
               <ul class="dropdown-menu">
                 <li>
-                  <a class="dropdown-item" href="/public/pages/games/">Jogos</a>
+                  <a class="dropdown-item" href="./public/pages/games/index.html">Jogos</a>
                 </li>
                 <li>
-                  <a class="dropdown-item" href="/public/pages/games/licoes_leitura.html">
+                  <a class="dropdown-item" href="./public/pages/games/licoes_leitura.html">
```

Comment:
Sugestão de correção:

```suggestion
                  <a class="dropdown-item" href="../games/licoes_leitura.html">
```